docs/build-on-linea/gas-fees.mdx

```diff
@@ -1,139 +1,108 @@
+# Estimate transaction costs
 
-As a Layer 2 blockchain, Linea provides a more stable and cost-effective solution for transaction fees. 
+Linea supports the [Ethereum EIP-1559 gas price model](https://ethereum.org/developers/docs/gas); however, being a
+layer 2 blockchain, Linea provides a more stable and cost-effective solution for transaction fees.
 
-**On average, gas fees on Linea are 15x cheaper than those on Ethereum.** Furthermore, we're continuously working towards reducing these costs even more to enhance affordability for our users.
+There are minor differences in the way Linea handles gas calculations to Ethereum:
 
-Linea is compatible with [EIP-1559](https://consensys.io/blog/what-is-eip-1559-how-will-it-change-ethereum); however, there are minor differences.
+- The base fee uses a set price of 7 wei. Blocks created by Linea use up to 30 million gas
+    (less than 50% of the maximum block size of 61 million gas), the fee decreases by 12.5% per block, effectively
+    keeping it at a stable 7 wei.
+- Transactions won't be sequenced if the `gasPrice` or `maxPriorityFeePerGas` falls below a certain threshold, which adjusts
+    over time. Instead, transactions are added to the pending queue until the gas price on the network drops sufficiently for the
+    transactions to be included.
 
-1.  On Linea, the base fee is **set at 7 wei, the minimum fee allowed by the Ethereum protocol after EIP-1559**. With all blocks created by Linea using up to 30M gas (less than 50% of the maximum block size of 61M gas), the fee decreases by 12.5% per block, effectively keeping it at a stable 7 wei.
-2.  Transactions won't be sequenced if the `gasPrice` or `maxPriorityFeePerGas` falls below a certain threshold, which adjusts over time.
+The gas cost to submit your transaction and include it on Ethereum involves the following fee components:
 
-# How do I make sure my transactions go through?
+- Layer 2 fee - The L2 fee (execution fee) is the cost to include your transaction on the Linea sequencer and is calculated
+    using a similar formula to Ethereum. 
+- Layer 1 fee - The L1 fee is the cost to publish your L2 transaction onto Ethereum and can vary based on the size
+    of the transaction calldata.
 
-### MetaMask Users
+You can [use the `linea_estimateGas`](../reference/api/linea-estimategas.mdx) gas API to estimate your gas costs; the result of the
+call provides an estimate that includes the layer 1 and layer 2 fees.
 
-**MetaMask should automatically calculate the necessary fees to ensure your transaction goes through.**
+## Calculate transaction fees
 
-### Non-MetaMask Users
+Use the [`linea_estimateGas`](../reference/api/linea-estimategas.mdx) API to estimate the gas cost for sending a
+transaction. The API will return the recommended gas limit, the base fee per gas, and the priority fee per gas.
```

Comment:
I would add a note here that we still support `eth_estimateGas`, `eth_gasPrice`, `eth_feeHistory` but they will usually return a higher value, hence why we recommend using `linea_gasPrice`